osu.Game/Localisation/GameplayMenuOverlayStrings.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Localisation;
+
+namespace osu.Game.Localisation
+{
+    public static class GameplayMenuOverlayStrings
+    {
+        private const string prefix = @"osu.Game.Resources.Localisation.GameplayMenuOverlay";
+
+        /// <summary>
+        /// "Continue"
+        /// </summary>
+        public static LocalisableString Continue => new TranslatableString(getKey(@"continue"), @"Continue");
+
+        /// <summary>
+        /// "Retry"
+        /// </summary>
+        public static LocalisableString Retry => new TranslatableString(getKey(@"retry"), @"Retry");
+
+        /// <summary>
+        /// "Quit"
+        /// </summary>
+        public static LocalisableString Quit => new TranslatableString(getKey(@"quit"), @"Quit");
+
+        /// <summary>
+        /// "failed"
+        /// </summary>
+        public static LocalisableString FailedHeader => new TranslatableString(getKey(@"failed_header"), @"failed");
+
+        /// <summary>
+        /// "paused"
+        /// </summary>
+        public static LocalisableString PausedHeader => new TranslatableString(getKey(@"paused_header"), @"paused");
+
+        /// <summary>
+        /// "you're dead, try again?"
+        /// </summary>
+        public static LocalisableString FailedDescription => new TranslatableString(getKey(@"failed_description"), @"you're dead, try again?");
+
+        /// <summary>
+        /// "you're not going to do what i think you're going to do, are ya?"
+        /// </summary>
+        public static LocalisableString PausedDescription => new TranslatableString(getKey(@"paused_description"), @"you're not going to do what i think you're going to do, are ya?");
```

Comment:
I'm not sure that we want to keep all of this strangely lowercase copy. Relevant: https://github.com/ppy/osu/pull/21765#issuecomment-1552378930, cc/ @Joehuu.

Reply:
I would be consistent and sentence case all the source strings and change the casing in code if deemed design appropriate.

Also if we have 2+ sentences, the [`ToSentence()`](https://github.com/ppy/osu-framework/blob/9f689c541b7c58459a63caf6487e36c82b34fd2b/osu.Framework/Localisation/CaseTransformableString.cs#L60-L68) method will not work currently as it just uppercases the first letter without regard to punctuation.

Reply:
Should I update code with sentence case?

Reply:
The `GameplayMenuOverlay` design is old, so it's probably fine to break the lowercase style for now, yes.

The other UI redesigns in figma seem to be transitioning out of that style, notably the song select footer.